### ISSUE TITLE
yarn seg faults on any command

### ISSUE THREAD

**Issue by 211217613** [yarnpkg/yarn#7294]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**
any yarn command seg faults

**If the current behavior is a bug, please provide the steps to reproduce.**

```
$ yarn --help
Segmentation fault (core dumped)
```

```
$ yarn --version
Segmentation fault (core dumped)
```

**What is the expected behavior?**

**Please mention your node.js, yarn and operating system version.**

```$ apt policy yarn
yarn:
  Installed: 1.16.0-1
  Candidate: 1.16.0-1
  Version table:
 *** 1.16.0-1 500
        500 https://dl.yarnpkg.com/debian stable/main amd64 Packages
        500 https://dl.yarnpkg.com/debian stable/main i386 Packages
        500 https://dl.yarnpkg.com/debian stable/main all Packages
        100 /var/lib/dpkg/status
```

```$ node -v
v10.15.2
```
```
$ cat /etc/os-release 
NAME="Ubuntu"
VERSION="19.04 (Disco Dingo)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 19.04"
VERSION_ID="19.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=disco
UBUNTU_CODENAME=disco
```

**Comment by konstantin-gusarov** [comment]
Update the Node.js to 10.16

**Comment by 211217613** [comment]
issue fixed itself

**Comment by japananh** [comment]
I faced the same issue. I tried to remove and reinstall node js and npm but it did not work. How did you fix it? My node version is 10.19.0

**Comment by japananh** [comment]
> I faced the same issue. I tried to remove and reinstall node js and npm but it did not work. How did you fix it? My node version is 10.19.0

I fixed it by updating to the lastest node js version 13.13.0
